package.json

```diff
@@ -9,6 +9,7 @@
     "preview": "vite preview"
   },
   "devDependencies": {
+    "@types/leaflet": "^1.9.8",
```

Comment:
Getting ready for something? :P Good job putting the types in the devDeps but just be aware that your PRs should be only about what they are about. This would be in the TS feature PR usually. No big deal considering it's only a dev dep.

Reply:
Thank you. The only reason I did this was because I was getting an error when importing L from leaflet and this was the suggsestion.

Reply:
Usually the whole `L.something()` approach is not how I handle leaflet stuff because it's a pain when L is not defined for whatever reason.
So if you're installing leaflet and not using the CDN, my recommendation is:
```ts
import { Circle } from 'leaflet';

// Note the capitalisation and new keyword as we are instantiating a class here
const circle = new Circle(lat, long);
```

as opposed to

```ts
const circle = L.circle(lat, long);
```